.git-blame-ignore-revs

```diff
@@ -0,0 +1,30 @@
+# Exclude these commits from git-blame and similar tools.
+#
+# To use this file, run the following command from the repo root:
+#
+# ```
+# $ git config blame.ignoreRevsFile .git-blame-ignore-revs
+# ```
+#
+# Include a brief comment with each commit added, for example:
+#
+# ```
+# 8f1152d4a22287a35be2dde596e3cf86ace8054a # Increase column limit to 100
+# ```
+#
+# Only add commits that are pure formatting changes (e.g. clang-format version changes, etc).
+
+# Apply formatting to thrust (#1616)
+165a06aa2c50b98d50fa56d027f0674da9d382e4
+
+# Apply formatting to libcudacxx (#1610)
+baee3f502d4fd5febc7f602428592b2bec5d2d3b
+
+# Apply clang-format to cub (#1602)
+eefcca862226672eae8eacb244f39f2e7b4bf7c4
+
+# Rework our system header approach to be more error proof (#661)
+774a2a552e9c42a9e5085e9aef2baa835daa2f69
```

Comment:
@alliepiper I also added some of the global refactoring commits here.

Those are not pure formatting changes, so we should discuss this internally

Reply:
@gevtushenko Also favors only adding formatting PRs